Configurations/windows-makefile.tmpl

```diff
@@ -471,11 +471,12 @@ clean: libclean
 	{- join("\n\t", map { "-del /Q /F $_" } @SCRIPTS) || "\@rem" -}
 	{- join("\n\t", map { "-del /Q /F $_" } @GENERATED_MANDATORY) || "\@rem" -}
 	{- join("\n\t", map { "-del /Q /F $_" } @GENERATED) || "\@rem" -}
-	-del /Q /S /F *.d *.obj *.pdb *.ilk *.manifest
+	-del /Q /S /F *.d *.obj *.pdb *.ilk *.manifest *.asm
```

Comment:
Aren't the generated .asm files already part of `@GENERATED` and therefore removed in line 473?